addons/hr_holidays/models/hr_leave_type.py

```diff
@@ -94,6 +93,15 @@ def _model_sorting_key(self, leave_type):
     support_document = fields.Boolean(string='Supporting Document')
     accruals_ids = fields.One2many('hr.leave.accrual.plan', 'time_off_type_id')
     accrual_count = fields.Float(compute="_compute_accrual_count", string="Accruals count")
+    # negative time off
+    allows_negative = fields.Boolean(string='Allow Negative Cap', default=False)
+    max_allowed_negative = fields.Integer(string="Amount in Negative")
+
+    _sql_constraints = [(
+        'check_negative',
+        'CHECK(NOT allows_negative OR max_allowed_negative > 0)',
+        'If negative amount are allowed, the maximum alloxed negative should be a value above 0..'
```

Comment:
I personally would allow 0 as well. I know it doesn't make too much sense to have this configuration, but it seems a bit overkill to explicitly throw an error for that.

Reply:
I think I had that discussion with Geoffrey, and it doesn't make sense. If the user wants no excess amount, they'll just untick the box

Reply:
Yes I know it does not make sense to have it. But I guarantee you at least some users are first going to have this option enabled, then change it back to 0 because they don't want it after all, and then spend about a minute to understand why they can't do what they want to do. While if we just allow it, it will be seamless for those users  without it really affecting the code or any flows in any way.